spider_feeder/store/file_store.py

```diff
@@ -53,10 +54,13 @@ def _open(self):
         return open(self._input_file_uri, encoding=self._file_encoding)
 
     def _parse(self, fd):
-        (_, file_extension) = path.splitext(self._input_file_uri)
-        file_extension = file_extension[1:]
-        logger.info(f'Parsing file {self._input_file_uri} with format {file_extension}.')
-        parser = load_object(self._parsers[file_extension])
+        file_format = self._file_format
```

Comment:
Why not moving this into a property called `file_format`?

Reply:
What do you mean, the `if` statement?

Reply:
Yeah, just to split the logic and specialize the functions.

Reply:
ok, it could be a method, but not a property.
I need the file name, which is not a class attribute